lib/customer_io.dart

```diff
@@ -70,4 +73,15 @@ class CustomerIO {
   static void setProfileAttributes({required Map<String, dynamic> attributes}) {
     return _customerIO.setProfileAttributes(attributes: attributes);
   }
+
+  /// Subscribes to an in-app event listener.
+  ///
+  /// [onEvent] - A callback function that will be called every time an in-app event occurs.
+  /// The callback returns [InAppEvent].
+  ///
+  /// Returns a [StreamSubscription] that can be used to subscribe/unsubscribe from the event listener.
+  static StreamSubscription subscribeToInAppEventListener(
```

Comment:
Is this function an async method? I am unsure from the function signature but I see `dark:async` module imported into the file making me think it is. 

If so, if there a reason for making this function async? I think the code runs synchronously?